dashboard/src/components/InstitutionLogo.vue

```diff
@@ -1,22 +1,24 @@
 <script setup lang="ts">
-const props = defineProps<{
+defineProps<{
   logo: string;
   name: string;
-  url: string;
+  url?: string;
 }>();
-
-const imgTag = `<img
-    id="institution-logo"
-    src="${props.logo}"
-    alt="${props.name}"
-    height="30"
-    
-/>`;
 </script>
 
 <template>
-  <div v-if="props.logo && props.url" class="d-none d-sm-block mx-3">
-    <a :href="props.url" target="_blank" rel="external" v-html="imgTag" />
+  <div v-if="logo" class="d-none d-sm-block mx-3">
+    <a v-if="url" :href="url" target="_blank" rel="external">
+      <img :src="logo" :alt="name" />
+    </a>
+    <img v-else :src="logo" :alt="name" />
   </div>
-  <div v-else-if="props.logo" class="d-none d-sm-block mx-3" v-html="imgTag" />
 </template>
+
+<style scoped>
+img {
+  max-height: 30px;
+  height: auto;
+  width: auto;
+}
+</style>
```

Comment:
Looks like this is the safe replacement for the old `height="30"`, so we don't have to write that twice.